packages/openapi-ts/src/utils/write/types.ts

```diff
@@ -355,10 +469,27 @@ const processServiceTypes = (client: Client, onNode: OnNode) => {
             ...emptyModel,
             export: 'any-of',
             isRequired: true,
-            properties: getSuccessResponses(operation.results),
+            properties: responses,
           }),
         });
 
+        if (
+          config.types.dates === 'types+transform' &&
+          responses.length === 1
```

Comment:
What should we do if there are multiple success responses? Do transformers need a way to discriminate the response based on status? Limiting support to 1 for now is safe

Reply:
Yeah, it will get more complex. This is an example moment where I'd love to have better logging so we could be aware of cases where we skip it and add them as we go... but I am sure people will come to Issues if they find something is not being handled. Maybe some kind of warning either in terminal or browser console would be helpful?

Reply:
That's a great idea, I'll add a warning for this situation with a link to come open an issue. That way we get case studies instead of guessing